### DISCUSSION TITLE
Vehicles

### DISCUSSION THREAD

**Discussion by lfxu** [godot-jolt/godot-jolt#741] [General]
I have a mostly-working vehicle support directly exposing Jolt vehicle constraints as additional nodes. I'm only a few days into Godot development so things may not be sanitized to Godot convensions, but if you are interested we may be able to work a PR out. Would that be something you would like to incorporate into godot-jolt?

**Comment by mihe** [comment]
Apologies for the late reply.

Due to things like #632 I'd rather focus on the drop-in aspect of this extension, meaning the existing physics API that Godot exposes, rather than adding the additional features/classes that Jolt offers.

So at this moment in time this is not something I'm interested in bringing in.
